go.mod

```diff
@@ -140,3 +140,5 @@ require (
 	sigs.k8s.io/structured-merge-diff/v4 v4.6.0 // indirect
 	sigs.k8s.io/yaml v1.4.0 // indirect
 )
+
+replace helm.sh/helm/v3 => helm.sh/helm/v3 v3.17.4 // fix CVE-2025-53547: use v3.17.4+ or v3.18.4+
```

Comment:
You ought to be able to bump it in the `indirect`. I don't think `replace` is necessary?